e2e-test/cypress/integration/overview_spec.js

```diff
@@ -25,7 +25,7 @@ describe("Overview Page", function() {
         ).to.contains.text("AR$ 32,000,000.00");
         expect(
           $card
-            .find("[data-test=project-creation]")
+            .find("[data-test=project-creation-date]")
             .children()
             .first()
```

Comment:
I think it is better to use cy.contains(content) instead of children(). cy.contains(content) yields the whole element and its children

Reply:
done